client/stylesheets/_responsive.scss

```diff
@@ -0,0 +1,105 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Sass Document For Adding Responsitivity 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+@media (min-width: 1200px) {
+    .oneDiv, .onediv {
+        max-width: 1024px !important;
+    }
+}
+
+@media (max-width: 1200px) {
+    .oneDiv, .onediv {
+        width: 90% !important;
+    }
+}
+
+@media (max-width: 1065px) {
+    .homeCardLi {
+    	width: 19.5%;
+    }
+}
+
+@media (max-width: 1000px) {
+    .homeCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `homeCardLi` should be written in lowercase with hyphens